src/System.Buffers.Experimental/System/Buffers/BufferExtensions.cs

```diff
@@ -218,7 +218,118 @@ private static unsafe int IndexOfVectorized(ref byte searchSpace, byte value, in
         exitFixed:;
                 return offset;
             }
+        }
+
+        public static bool TryIndicesOf(this Span<byte> buffer, byte value, Span<int> indices, out int numberOfIndices)
```

Comment:
You need both Span and ReadOnlySpan. I have both. I just mistyped the name of one of the methods.